src/Ssh/Ssh/Common/SshResourceIdCompleterAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.PowerShell.Cmdlets.Ssh.Common
+{
+    class SshResourceIdCompleterAttribute : ArgumentCompleterAttribute
```

Comment:
In PowerShell code base, we are always explicit about type accessiblity.
```suggestion
    private class SshResourceIdCompleterAttribute : ArgumentCompleterAttribute
```

Reply:
When I do this change I get an error:
"Elements defined in a namespace cannot be explicitly declared as private, protected, protected internal, or private protected."

Reply:
Sorry, I guess this would be `internal`.  I assume it is not a type intended for public use?